src/test/java/com/adyen/TerminalCloudAPITest.java

```diff
@@ -189,4 +189,23 @@ public void syncAbortRequestSuccess() throws Exception {
 
         assertNull(terminalAPIResponse);
     }
+
+    /**
+     * Test success flow for Input Request on POST /sync
+     */
+    @Test
+    public void syncInputRequestSuccess() throws Exception {
+        Client client = createMockClientFromFile("mocks/terminal-api/input-request-success.json");
+        TerminalCloudAPI terminalCloudApi = new TerminalCloudAPI(client);
+
+        TerminalAPIResponse requestResponse = terminalCloudApi.sync(new TerminalAPIRequest());
+
+        assertNotNull(requestResponse);
```

Comment:
Is the success for this test determined by the fact that none of the properties are null ?

Reply:
The point is to check if we get a MenuEntryNumber.
I usually like to also add null checks before that, but I can remove if you guys feel it's not needed.

Reply:
Nah was just checking, it's alright for me. It can be useful if we ever move to a different source instead of the mocked ones.